examples/Protobuf-legacy/cmake/Hunter/config.cmake

```diff
@@ -0,0 +1,14 @@
+if(ANDROID OR IOS)
+  hunter_config(Protobuf
+      VERSION ${HUNTER_Protobuf_VERSION}
+      CMAKE_ARGS
+      protobuf_BUILD_PROTOC=OFF
+      protobuf_BUILD_LIBPROTOBUF_LITE=OFF
+  )
+else()
+  hunter_config(Protobuf
+      VERSION ${HUNTER_Protobuf_VERSION}
+      CMAKE_ARGS
+      protobuf_BUILD_LIBPROTOBUF_LITE=OFF
```

Comment:
We don't need it in the example, usually you only need either the full or lite version.
I guess hunter defaults should still be to build both, so there won't be 2 different installs. 
I skipped the lite library here just to improve build time and show this feature in the example.